h/models/group.py

```diff
@@ -126,6 +126,13 @@ def type(self):
     def is_public(self):
         return self.readable_by == ReadableBy.world
 
+    @property
+    def logo(self):
+        """Returns logo icon if one exists for this groups authority else
+        returns None."""
+        return {'biopub.hypothes.is': 'biopub-logo',
```

Comment:
BioPub as currently set up at https://biopub.hypothes.is does not use a custom authority. They are designed to be representative of most of our anticipated initial users of the new group types in that they use Hypothesis accounts.